### DISCUSSION TITLE
Dual Screen

### DISCUSSION THREAD

**Discussion by Guerny** [valinet/ExplorerPatcher#1129] [Ideas]
First, I would like to add my thanks to valinet for creating this Explorer Patcher / Windows 10 taskbar replacement to Windows 11 taskbar.  
Well done - I have just upgraded from Win 7 (having looked at Win 8 and been horrified, and not bothered with Win10) and can't believe how Microsoft has botched things.  I have used the taskbar at the top of the screen all my life since windows started and could not use it at the bottom of the screen - such a lot of extra effort and waste of time!  
Much else to learn, I'm afraid.

Second does this program work with a dual screen?  I can get one taskbar to the top of my main screen, but can't figure out how to get the taskbar on the second screen to the top.  Am I Missing something?

any help much appreciated.
Guerny

**Comment by valinet** [comment]
Yeah, right click taskbar, uncheck Lock all taskbars, drag desired taskbar to the top of the screen.

**Reply by PandaBoyo** [reply]
Hi valinet, figured I'd reply here rather than make a whole new thread since my question's sort of in the same vein. Would it be possible to add more customization to the default windows 11 taskbar like there is for the windows 11 one, like changing the taskbar position? Or would it be possible to keep certain aspects of the default windows 11 taskbar while using the windows 10 taskbar, such as the combined calendar/notifications button and the combined networks/accessibility/volume button? Thanks.

**Comment by Guerny** [comment]
Many many thanks, Valinet. So simple to get both taskbars to the top! 
Makes me feel dumb for missing it!
Guerny

**Comment by TossOne53** [comment]
SHOULD ExplorerPatcher taskbar location work on more than two screens?  I have my taskbar on the left on Identify screens 1 and 3 but screen 2 does not participate.  Screen 2 is connected via Dpin, screens 1 and 3 via HDMI.  Running Win11 Home build 26100 and EP 26100.4946.69.4
